Cargo.toml

```diff
@@ -15,16 +15,20 @@ crate-type = ["cdylib", "lib"]
 [dependencies]
 ahash = "0.8.11"
 bincode = "1.3.3"
-napi = { version = "2.16.16", default-features = false, features = [
-    "napi8",
-    "serde-json",
+napi = { version = "3.0.0-alpha.29", default-features = false, features = [
```

Comment:
Any reason you are using alpha version here? This is a stable library, hoping to avoid living on the edge.